code/modules/mob/living/carbon/life.dm

```diff
@@ -650,11 +650,11 @@ All effects don't start immediately, but rather get worse over time; the rate is
 	var/temp_diff = bodytemperature - M.bodytemperature
 	if(temp_diff > 0) // you are warm share the heat of life
 		M.adjust_bodytemperature(temp_diff, use_insulation=TRUE, use_steps=TRUE) // warm up the giver

```

Comment:
oh actually this should be 1/2 temp_diff for all these or you'll just swap temperatures between mobs every hug

and maybe capped to some value and also maybe influenced by clothing/heat/cold protection

Reply:
That's beyond the scope of this PR. It also won't just keep swapping it because all races have different base temps. I'm adjusting one equation on something that was merged within the last month. I'm not doing anymore.

Reply:
`use_insulation=TRUE` takes heat and cold protection into account

Reply:
You know that is what the use steps argument does right?